services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HankeFactory.kt

```diff
@@ -325,5 +374,17 @@ class HankeFactory(
             id: Int,
             mutator: (HankeYhteystieto) -> Unit = {}
         ): Hanke = withGeneratedToteuttajat(listOf(id), mutator)
+
+        private fun tormaysTarkastelu(
+            id: Int = 1,
+            hankeEntity: HankeEntity
+        ): TormaystarkasteluTulosEntity =
+            TormaystarkasteluTulosEntity(
+                id = id,
+                perus = Random.nextFloat(),
+                pyoraily = Random.nextFloat(),
+                joukkoliikenne = Random.nextFloat(),
```

Comment:
Using randomized values in unit tests is a bad idea. Unit tests should be repeatable and test the same thing every time. Using random values also means tests can break randomly, like in the CI pipeline.

The random values here are also outside the defined value range of the indexes. They're values between 1 and 5, when the random values are between 0 and 1.

Fuzz testing can be done separately if needed.